apps/api/src/models/ApplicationData.py

```diff
@@ -85,7 +87,7 @@ class RawMentorApplicationData(BaseMentorApplicationData):
     first_name: str
     last_name: str
     resume: Union[UploadFile, None] = None
```

Comment:
Question: for mentors, I see the resume is currently specified as required. If that is the case, then I think this field should not be possibly `None`.